src/Npgsql/TypeMapping/GlobalTypeMapper.cs

```diff
@@ -123,6 +130,12 @@ internal NpgsqlDbType ToNpgsqlDbType(DbType dbType)
         internal DbType ToDbType(Type type)
             => _typeToDbType.TryGetValue(type, out var dbType) ? dbType : DbType.Object;
 
+        internal string ToPgTypeName(NpgsqlDbType npgsqlDbType)
+            => _npgsqlDbTypeToPgTypeName.TryGetValue(npgsqlDbType, out var pgTypeName) ? pgTypeName : "";
+
+        internal string ToPgTypeName(Type type)
+            => _typeToPgTypeName.TryGetValue(type, out var pgTypeName) ? pgTypeName : "";
```

Comment:
Seems better to have DataTypeName return null when the type name cannot be found. With NRT, this would force people to check for that case etc.

Reply:
May I resolve this conversation (and others that have already been addressed)? I'm not sure how is your workflow regarding this